Source/Core/Core/ConfigLoaders/BaseConfigLoader.cpp

```diff
@@ -17,7 +17,7 @@
 #include "Common/FileUtil.h"
 #include "Common/IniFile.h"
 #include "Common/Logging/Log.h"
-#include "Common/SysConf.h"
+#include "Core/SysConf.h"
```

Comment:
Thoughts on moving it further down to the Core group below?

Reply:
Whoops. Fixed!